### PR TITLE
Responsive Design for User Profile

### PR DESCRIPTION
**Pull request by corwinharrell**
- Restructure of profile bounties section to more closely resemble and function like explorer
- Responsive design for profile

screen shot 2018-08-23 at 5 13 07 pm

### CODE REVIEW

File: src/containers/Profile/Profile.module.scss [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+  height: 100%;
 }
 
 .profileDetails {
   background: $brand-white;
 }
 
-.profileBounties {
-  background: $brand-nearWhite;
+.profileBountiesContainer {
```
Comment (villanuevawill) [inline src/containers/Profile/Profile.module.scss]: much better :)

File: src/containers/Profile/ProfileBounties.module.scss [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+
+  @media only screen and (min-width: 48em) {
+    padding: $xl-spacing $base-spacing $xl-spacing $base-spacing;
+    width: calc(100% - #{$filterNav-width-fluid});
+  }
+
+  @media only screen and (min-width: 65em) {
+    padding-left: $m-spacing;
+    padding-right: $m-spacing;
+    width: calc(100% - #{$filterNav-width-fixed});
```
Comment (villanuevawill) [inline src/containers/Profile/ProfileBounties.module.scss]: Is calcul compatibe everywhere?

Comment (villanuevawill) [reply src/containers/Profile/ProfileBounties.module.scss]: *calc compatible

File: src/containers/Profile/ProfileBounties.module.scss [hunk 3]
```diff
@@ -11,6 +25,11 @@
   flex-direction: column;
 }
 
+.bodyInnerContainer {
+  max-width: 1040px;
```
Comment (villanuevawill) [inline src/containers/Profile/ProfileBounties.module.scss]: So no projector support @corwinharrell :)

Comment (corwinharrell) [reply src/containers/Profile/ProfileBounties.module.scss]: derp I meant to change this to `rem`

File: src/containers/Profile/ProfileBounties.module.scss (hunk 2 above)
Comment (corwinharrell) [reply src/containers/Profile/ProfileBounties.module.scss]: yep! https://caniuse.com/#feat=calc

**Pull request merged**